test/logs/log_utils.ts

```diff
@@ -45,7 +45,19 @@ export function formatLogEntry(logs: ABIDecoder.DecodedLog): Log {
   // Loop through each event and add to args
   _.each(events, (event) => {
     const { name, type, value } = event;
-    args[name] = (/^(uint)\d*/.test(type)) ? new BigNumber(value.toString()) : value;
+
+    var argValue: any = value;
+    switch (true) {
+      case (/^(uint)\d*\[\]/.test(type)): {
+        break;
```

Comment:
Does this work? Don't we want to set the `argValue` to be an array of BigNumbers if this is the case?

Reply:
Yup, it worked for me